### PR TITLE
Add networking_utils pack

### PR DESCRIPTION
**Pull request by jjm**
### Status 

- new pack, complete

### Description

Add a new pack containing some useful networking valuation actions for use in workflows. 

### Checklist (tick everything that applies)

- [X] Metadata: pack.yaml, icon, structure (required for new packs)
- [X] Version bump (required for changed packs)
- [X] Code linting (required, can be done after the PR checks)
- [X] [Tests](https://docs.stackstorm.com/development/pack_testing.html) (not required but really recommended)
- [X] Installs via GitHub with packs deploy.

### CODE REVIEW

**Comment by Kami** [comment]
Thanks!

For consistency with other pack names, please use underscore instead of - in the directory name (`-` won't work in all the places since we actually use directory name for pack reference right now in some places and not the actual `name` attribute from `pack.yaml`)

File: packs/networking-utils/actions/is_valid_asa_nameif.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from st2actions.runners.pythonrunner import Action
+
+
+class IsValidNameifAction(Action):
+    def run(self, nameif):
+        """
+        Checks if nameif is valid for use on a Cisco ASA as an interface name.
```
Comment (Kami) [inline packs/networking-utils/actions/is_valid_asa_nameif.py]: I'm fine with this action living in this pack, but I would imagine some people would probably prefer it to be in a separate `cisco` pack :P

**Comment by jjm** [comment]
That's it renamed, as requested.

File: packs/networking-utils/actions/is_valid_asa_nameif.py (hunk 1 above)
Comment (jjm) [inline packs/networking-utils/actions/is_valid_asa_nameif.py]: True, it maybe it should be in a Cisco pack. However seems to fit within the validation idea behind this pack.

If others object I'm happy to remove for now.

**Comment by jjm** [comment]
I've confirmed manually they all load with the right pack name and work within ST2 ok.

**Comment by Kami** [comment]
Merged, thanks!

If people are not happy with cisco util action being in this pack, we can always move it later. I will also add an icon in a separate commit.

**Comment by manasdk** [comment]
Awesome!

**Pull request merged**